CS/switcher/switcher/ThemeSwitcher/ThemeSwitcher.razor

```diff
@@ -39,7 +43,11 @@
         }
     }
     protected override void OnInitialized() {
-        ThemeName = Themes.ActiveTheme?.Name ?? string.Empty;
+        _persistingSubscription = ApplicationState.RegisterOnPersisting(PersistData);
+        if (ApplicationState.TryTakeFromJson<string>("ThemeName", out var themeName) && themeName != null)
```

Comment:
Is it possible that a value is an empty string? Maybe it's better to use the `IsNullOrEmpty` method

Reply:
I think not, because we get the same value we set before. I added the null check just to avoid a compile warning because the return value is nullable.